metadata/config.yaml

```
@@ -107,49 +132,73 @@ DEPOSIT_CONTRACT_ADDRESS: 0x00000000219ab540356cBB839Cbe05303d7705Fa
 
 # Networking
 # ---------------------------------------------------------------
-# `10 * 2**20` (= 10485760, 10 MiB)
+# 10 * 2**20 (= 10,485,760) bytes, 10 MiB
 MAX_PAYLOAD_SIZE: 10485760
-# `2**10` (= 1024)
+# 2**10 (= 1,024) blocks
 MAX_REQUEST_BLOCKS: 1024
-# `2**8` (= 256)
+# 2**8 (= 256) epochs
 EPOCHS_PER_SUBNET_SUBSCRIPTION: 256
-# `MIN_VALIDATOR_WITHDRAWABILITY_DELAY + CHURN_LIMIT_QUOTIENT // 2` (= 33024, ~5 months)
+# MIN_VALIDATOR_WITHDRAWABILITY_DELAY + CHURN_LIMIT_QUOTIENT // 2 (= 33,024) epochs
 MIN_EPOCHS_FOR_BLOCK_REQUESTS: 33024
-# 5s
-TTFB_TIMEOUT: 5
-# 10s
-RESP_TIMEOUT: 10
+# 2**5 (= 32) slots
 ATTESTATION_PROPAGATION_SLOT_RANGE: 32
 # 500ms
 MAXIMUM_GOSSIP_CLOCK_DISPARITY: 500
 MESSAGE_DOMAIN_INVALID_SNAPPY: 0x00000000
 MESSAGE_DOMAIN_VALID_SNAPPY: 0x01000000
 # 2 subnets per node
 SUBNETS_PER_NODE: 2
-# 2**8 (= 64)
+# 2**6 (= 64) subnets
 ATTESTATION_SUBNET_COUNT: 64
+# 0 bits
 ATTESTATION_SUBNET_EXTRA_BITS: 0
-# ceillog2(ATTESTATION_SUBNET_COUNT) + ATTESTATION_SUBNET_EXTRA_BITS
+# ceillog2(ATTESTATION_SUBNET_COUNT) + ATTESTATION_SUBNET_EXTRA_BITS (= 6 + 0) bits
 ATTESTATION_SUBNET_PREFIX_BITS: 6
 
 # Deneb
-# `2**7` (=128)
+# 2**7 (= 128) blocks
 MAX_REQUEST_BLOCKS_DENEB: 128
-# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK
-MAX_REQUEST_BLOB_SIDECARS: 768
-# `2**12` (= 4096 epochs, ~18 days)
+# 2**12 (= 4,096) epochs
 MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 4096
-# `6`
+# 6 subnets
 BLOB_SIDECAR_SUBNET_COUNT: 6
+# 6 blobs
+MAX_BLOBS_PER_BLOCK: 6
+# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK (= 128 * 6) sidecars
+MAX_REQUEST_BLOB_SIDECARS: 768
 
 # Electra
-# 2**7 * 10**9 (= 128,000,000,000)
-MIN_PER_EPOCH_CHURN_LIMIT_ELECTRA: 128000000000
-# 2**8 * 10**9 (= 256,000,000,000)
-MAX_PER_EPOCH_ACTIVATION_EXIT_CHURN_LIMIT: 256000000000
-# `9`
+# 9 subnets
 BLOB_SIDECAR_SUBNET_COUNT_ELECTRA: 9
-# `uint64(9)`
+# 9 blobs
 MAX_BLOBS_PER_BLOCK_ELECTRA: 9
-# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK_ELECTRA
+# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK_ELECTRA (= 128 * 9) sidecars
 MAX_REQUEST_BLOB_SIDECARS_ELECTRA: 1152
+
+# Fulu
+# 2**7 (= 128) groups
+NUMBER_OF_CUSTODY_GROUPS: 128
+# 2**7 (= 128) subnets
+DATA_COLUMN_SIDECAR_SUBNET_COUNT: 128
+# MAX_REQUEST_BLOCKS_DENEB * NUMBER_OF_COLUMNS (= 128 * 128) sidecars
+MAX_REQUEST_DATA_COLUMN_SIDECARS: 16384
+# 2**3 (= 8) samples
+SAMPLES_PER_SLOT: 8
+# 2**2 (= 4) sidecars
+CUSTODY_REQUIREMENT: 4
+# 2**3 (= 8) sidecars
+VALIDATOR_CUSTODY_REQUIREMENT: 8
+# 2**5 * 10**9 (= 32,000,000,000) Gwei
+BALANCE_PER_ADDITIONAL_CUSTODY_GROUP: 32000000000
+# 2**12 (= 4,096) epochs
+MIN_EPOCHS_FOR_DATA_COLUMN_SIDECARS_REQUESTS: 4096
+
+
+# Blob Scheduling
+# ---------------------------------------------------------------
+
+BLOB_SCHEDULE:
+  - EPOCH: 414464 # December 17, 2025, 01:29:59pm UTC
+    MAX_BLOBS_PER_BLOCK: 15
+  - EPOCH: 419072 # January 7, 2026, 01:01:11am UTC
```

There should be two spaces between content & comment.
```suggestion
  - EPOCH: 414464  # December 17, 2025, 01:29:59pm UTC
    MAX_BLOBS_PER_BLOCK: 15
  - EPOCH: 419072  # January 7, 2026, 01:01:11am UTC
```